### PR TITLE
Events history and csv

### PR DESCRIPTION
**Pull request by AmsTaFFix**

### CODE REVIEW

File: src/Model/BouncesGetResponse/Bounce.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    /**
+     * Get bounce type
+     *
+     * @return BounceType Bounce type
+     */
+    public function getBounceType(): BounceType
+    {
+        return $this->bounceType;
+    }
+
```
Comment (liderman) [inline src/Model/BouncesGetResponse/Bounce.php]: Please remove empty line

File: src/Response/SubscribersGetEventsHistoryResponse.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * Subscriber's history of events
+ *
+ * @author Nikita Sapogov <sapogov.n@citilink.ru>
+ */
+class SubscribersGetEventsHistoryResponse extends SpecificXmlMethodResponse
+{
+    /**
+     * Get events
+     *
```
Comment (liderman) [inline src/Response/SubscribersGetEventsHistoryResponse.php]: Specify in the documentation that the method can throw exceptions

File: src/SpecificCsvMethodResponse.php [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+     * @inheritdoc
+     */
+    public function getErrorMessages(): array
+    {
+        return $this->response->getErrorMessages();
+    }
+
+    /**
+     * Get csv line values without header
+     *
```
Comment (liderman) [inline src/SpecificCsvMethodResponse.php]: Specify in the documentation that the method can throw exceptions

File: tests/ExpertSenderTest.php [hunk 4]
```diff
@@ -215,6 +215,11 @@ public function testChangeEmail()
         $this->assertEquals(400, $subscriberInfoByOldEmail->getErrorCode());
     }
 
+    public function testBouncesGet()
```
Comment (liderman) [inline tests/ExpertSenderTest.php]: Why this test? He does not do anything at all

File: src/Response/SubscribersGetEventsHistoryResponse.php (hunk 2 above)
Comment (AmsTaFFix) [inline src/Response/SubscribersGetEventsHistoryResponse.php]: fixed

File: src/SpecificCsvMethodResponse.php (hunk 3 above)
Comment (AmsTaFFix) [inline src/SpecificCsvMethodResponse.php]: fixed

**Pull request merged**